ports/di/vcpkg.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "boost-di",
+  "name": di",
```

```suggestion
  "name": "di",
```

Missing an opening `"`.

---

I see, however, it can't have boost in the name, what about `dependency-injection` then?

---

How about bext-di?

---

I'm okay with bext-di :+1:

---

@kreuzerkrieg, could you please update the port name to 'bext-di' and continue this PR?

---

BTW,  you can format the vcpkg.json file by command './vcpkg format-manifest' ./ports/bext-di/vcpkg.json.

---

hehe, how come my name suddenly popped up here? :)
will take a look at the evening

---

Sorry for my mistakes!